__tests__/server/utils/pollModuleMap.spec.js

```diff
@@ -284,11 +279,11 @@ describe('pollModuleMap', () => {
     // check that the interval has increased
     expect(setTimeout.mock.calls[0][1]).toBeGreaterThanOrEqual(MIN_POLL_TIME * 1.25);
 
-    expect(setGauge).toHaveBeenCalledTimes(1);
-    expect(setGauge.mock.calls[0][0]).toBe(holocronMetrics.moduleMapPollWait);
-    expect(setGauge.mock.calls[0][1]).toBeGreaterThanOrEqual(5 * 1.25);
+    expect(setGauge).toHaveBeenCalledTimes(3);
+    expect(setGauge.mock.calls[2][0]).toBe(holocronMetrics.moduleMapPollWait);
+    expect(setGauge.mock.calls[2][1]).toBeGreaterThanOrEqual(5 * 1.25);
     // ensure we're using seconds, not milliseconds
-    expect(setGauge.mock.calls[0][1]).toBeLessThan(8 * 1.25);
+    expect(setGauge.mock.calls[2][1]).toBeLessThan(8 * 1.25);
```

Comment:
why are we multiplying by 1.25?

Reply:
we increase the poll time by `*1.25` each cycle - 
https://github.com/americanexpress/one-app/pull/1131/files#diff-aee22318f291692511a5103c699287cfeb2d61fbc7b3ae9d8395019a73c947d9R62